### ISSUE TITLE
Favicons messageslist

### ISSUE THREAD

**Issue by caprihomeachim** [thunderbird/thunderbird-android#6657]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.507

### Problem you are trying to solve

it would be nice if a favicon of the respective domain was displayed before the messages in the messageslist instead of a contact picture.

### Suggested solution

add favicons

### Screenshots / Drawings / Technical details
![Screenshot_20230210-194454](https://user-images.githubusercontent.com/125082387/218173087-bc115417-22b6-4cb2-8102-60d7c6c9c5c1.png)
_No response_

**Comment by ByteHamster** [comment]
I guess most people already know this, but because it was not mentioned in this issue yet: This feature is what got FairEmail kicked out of the Google Play Store last year

**Comment by caprihomeachim** [comment]
No, it is not the whole history, it came back to the play store, and still with the favicons.

**Comment by caprihomeachim** [comment]
Blue mail and emclient are another email apps with favicons available in the google play store!

**Comment by caprihomeachim** [comment]
Not any news about this?

**Comment by kewisch** [comment]
This could possibly be done via https://datatracker.ietf.org/doc/draft-brand-indicators-for-message-identification/ but there are a few additional requirements in there that might not be easy to implement. I'm not opposed to this, if someone wants to work on this I'd appreciate some planning prior to diving into the code given the size of the feature and potential privacy implications.

We've recently started moving feature requests over to [Mozilla Connect](https://connect.mozilla.org/t5/ideas/idb-p/ideas/label-name/thunderbird%20android), would you mind re-posting the idea there?

To get the most out of your post, check out [this quick guide](https://connect.mozilla.org/t5/discussions/how-to-submit-a-great-idea-in-five-easy-steps/m-p/24) on crafting impactful suggestions. Focusing on the problem your idea solves and how it could help others can really make it stand out!
